app/screens/edit_post/edit_post.tsx

```diff
@@ -269,22 +221,18 @@ const EditPost = ({componentId, maxPostSize, post, closeButtonId, hasFilesAttach
                     />
                 </View>
             </SafeAreaView>
-            <Animated.View style={animatedStyle}>
-                <Autocomplete
-                    channelId={post.channelId}
-                    hasFilesAttached={hasFilesAttached}
-                    nestedScrollEnabled={true}
-                    rootId={post.rootId}
-                    updateValue={onChangeText}
-                    value={postMessage}
-                    cursorPosition={cursorPosition}
-                    postInputTop={1}
-                    fixedBottomPosition={true}
-                    maxHeightOverride={isTablet ? 200 : undefined}
-                    inPost={false}
-                />
-            </Animated.View>
-
+            <Autocomplete
```

Comment:
just for my understanding, why is the animation being removed?

Reply:
If I am not mistaken, it was not doing much. Also, meddled with the absolute positioning of the Autocomplete (absolute positioning is relative to the parent, so in this case the animated view).